src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -205,6 +205,9 @@ export interface ITerminalConfiguration {
 	bellDuration: number;
 	defaultLocation: TerminalLocationString;
 	customGlyphs: boolean;
+	title: string;
+	description: string;
+	separator: string;
```

Comment:
Shall we move these to `.tabs`?

Reply:
they also apply to the editor terminals, so I don't think so